lib/barkibu-kb.rb

```diff
@@ -17,6 +17,11 @@ module KB
   end
 
   setting :log_level, default: :info
+
+  setting :request do
+    setting :open_timeout, default: 5
+    setting :timeout, default: 5
```

Comment:
why not leave only one option - `timeout`?

Reply:
According to [docs](https://lostisland.github.io/faraday/#/customization/request-options?id=request-options), I'm unclear if `timeout` includes both connection establishment and response completion or it does not count connection.
If you are sure about the behaviour, we can leave just `timeout`.

Reply:
net/http doesn't have `timeout` config. It only has 3 specific timeouts(open, read and write). So setting `timeout` on faraday's level - will set all 3 specific timeouts to 5 on net/http level. https://github.com/lostisland/faraday/blob/main/lib/faraday/adapter.rb#L75-L83

So with this in mind - I prefer to either expose all 3 specific timeout config options from the kb gem or just leave one - `timeout`